docs/website/docs/dlt-ecosystem/file-formats/parquet.md

```diff
@@ -76,3 +77,16 @@ You can generate parquet files without timezone adjustment information in two wa
 2. Set the **timestamp_timezone** to empty string (ie. `DATA_WRITER__TIMESTAMP_TIMEZONE=""`) to generate logical type without UTC adjustment.
 
 To our best knowledge, arrow will convert your timezone aware DateTime(s) to UTC and store them in parquet without timezone information.
+
+
+### Row group size
+`pyarrow` parquet writer writes each item (be it table or batch which is internally converted to table) in a separate row group. This may lead to many small row groups
+to be created which may not be optimal for certain query engines (ie. `duckdb` parallelizes on a row group). `dlt` allows to control the size of the row group by
+buffering and concatenating (0 copy) tables/batches before they are written. You can control the size by setting the buffer size
+```toml
+[extract.data_writer]
+buffer_max_items=1000000
+```
+Mind that we must hold the tables in memory. 1 000 000 rows in example above may take quite large amount of it.
+
+`row_group_size` has limited utility with `pyarrow` writer. It will split large tables into many groups if set below item buffer size.
```

Comment:
This sentence is not clear to me yet. Is the instruction to the user something like the following?

```suggestion
For the `pyarrow` parquet writer, ensure to have`row_group_size >= buffer_max_items`. Otherwise, your destination might have more row groups than optimal.
```

Reply:
ohhh so actually the reverse is true. `row_group_size < buffer_max_item` to have any effect. this is the core of the problem I'm fixing here. pyarrow will create row group of size of parquet table being written or smaller. btw. other, well desgined implementations allow to write batches to the same groups. not here